build/dependencies.props

```diff
@@ -12,7 +12,7 @@
     <MicrosoftAspNetCoreAuthorizationPackageVersion>2.1.0-preview2-30451</MicrosoftAspNetCoreAuthorizationPackageVersion>
     <MicrosoftAspNetCoreAuthorizationPolicyPackageVersion>2.1.0-preview2-30451</MicrosoftAspNetCoreAuthorizationPolicyPackageVersion>
     <MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>2.1.0-preview2-30451</MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>
-    <MicrosoftAspNetCoreConnectionsAbstractionsPackageVersion>2.1.0-preview2-30451</MicrosoftAspNetCoreConnectionsAbstractionsPackageVersion>
+    <MicrosoftAspNetCoreConnectionsAbstractionsPackageVersion>2.1.0-a-preview2-remove-duplex-pipe-17616</MicrosoftAspNetCoreConnectionsAbstractionsPackageVersion>
```

Comment:
Test this doesn't break VS build.

Reply:
LOL 😄

Reply:
![image](https://user-images.githubusercontent.com/95136/37955396-56293960-315d-11e8-9b8d-30df7d112018.png)

Reply:
Are we going to have to go back to the days of yore, where we had to have our code checkouts in `C:\dd` in order to keep paths short? 😛